EasyPost/Address.cs

```diff
@@ -204,27 +169,27 @@ public static async Task<AddressCollection> All(Dictionary<string, object>? para
             return await request.Execute<AddressCollection>();
         }
 
-        private static async Task<Address> SendCreate(Dictionary<string, object> parameters, List<string>? verifications = null,
-            List<string>? strictVerifications = null)
```

Comment:
Please callout the fact that the `verifications` and `strictVerifications` params are being removed from this interface in favor of the better documented `parameters` workflow in the CHANGELOG.